packages/block-library/src/post-author-name/block.json

```diff
@@ -27,7 +27,11 @@
 		"html": false,
 		"spacing": {
 			"margin": true,
-			"padding": true
+			"padding": true,
+			"__experimentalDefaultControls": {
+				"margin": false,
+				"padding": false
+			}
```

Comment:
Was there a reason this PR changed the spacing controls to be optional and hidden by default?

If spacing and border controls were being added new via this PR, I'd agree with them both being optional however the spacing support on this block has been in for a while and displaying the controls by default.

I think it might be better to leave that as it is and simply make the border support's controls match.

Reply:
Thanks for the feedback.

No, there's no reason. I thought spacing control should be optional, But I Reverted the changes and kept spacing control as the default.

Reply:
My understanding is that the border controls should match the dimension and spacing controls. So if the spacing controls are already displayed by default, then so should border controls. 

For this PR specifically, the spacing controls were displayed by default so they should remain so as you've already fixed. Now the border controls should also be made to be displayed by default and match the spacing controls.

Reply:
Thank you for the update. Border controls will now be displayed by default. (Updated Changes in the Latest Commit.)

I will take care of this point in the upcoming PR. (The dimensions of the border controls now match those of the spacing controls.)